compendium/DeclarativeServices/src/manager/ConfigurationNotifier.hpp

```diff
@@ -76,7 +77,8 @@ namespace cppmicroservices
              */
             ConfigurationNotifier(cppmicroservices::BundleContext const& context,
                                   std::shared_ptr<cppmicroservices::logservice::LogService> logger,
-                                  std::shared_ptr<cppmicroservices::async::AsyncWorkService> asyncWorkService_);
+                                  std::shared_ptr<cppmicroservices::async::AsyncWorkService> asyncWorkService_,
+                                  std::shared_ptr<SCRExtensionRegistry> extensionRegistry_);
```

Comment:
the parameter names have the trailing `_` which doesn't fit in with the convention where the members have the trailing `_`. I recommend putting the trailing `_` on the member variable name.